hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -331,6 +342,116 @@ final Stream<DynamicTest> chargeAssociationFeeForNFT() {
+                final String NFT_FOR_MULTIPLE_PENDING_TRANSFER = "nftForMultiplePendingTransfer";
+                final String FT_FOR_MULTIPLE_PENDING_TRANSFER = "ftForMultiplePendingTransfer";
+                var nftSupplyKeyForMultipleTransfers = "nftSupplyKeyForMultipleTransfer";
+                return defaultHapiSpec("Send multiple FT and NFT from EOA to Account without free Auto-Associations")
+                        .given(
+                                tokenCreate(FT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                        .treasury(OWNER)
+                                        .tokenType(FUNGIBLE_COMMON)
+                                        .initialSupply(1000L),
+                                newKeyNamed(nftSupplyKeyForMultipleTransfers),
+                                tokenCreate(NFT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                        .treasury(OWNER)
+                                        .tokenType(NON_FUNGIBLE_UNIQUE)
+                                        .initialSupply(0L)
+                                        .name(NFT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                        .supplyKey(nftSupplyKeyForMultipleTransfers),
+                                mintToken(
+                                        NFT_FOR_MULTIPLE_PENDING_TRANSFER,
+                                        IntStream.range(0, 10)
+                                                .mapToObj(a -> ByteString.copyFromUtf8(String.valueOf(a)))
+                                                .toList()))
+                        .when(
+                                tokenAirdrop(
+                                                moving(10, FT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS),
+                                                movingUnique(NFT_FOR_MULTIPLE_PENDING_TRANSFER, 1L)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS))
+                                        .payingWith(OWNER)
+                                        .signedBy(OWNER)
+                                        .via("first airdrop"),
+                                tokenAirdrop(
+                                                moving(10, FT_FOR_MULTIPLE_PENDING_TRANSFER)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS),
+                                                movingUnique(NFT_FOR_MULTIPLE_PENDING_TRANSFER, 2L)
+                                                        .between(OWNER, RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS))
+                                        .payingWith(OWNER)
+                                        .signedBy(OWNER)
+                                        .via("second airdrop"))
```

Comment:
I think that you don't need the second airdrop transaction, the test states one token airdrop with multiple tokens so no need for second airdrop transaction.

Reply:
While I created this test, we commented with the colleagues that it would be good to have the scenario with consequent airdrops with multiple transfers in order to check that the amount for the fungible token is aggregated into only one airdrop and the pending airdrops for the non-fungible tokens are separate and the association fees are prepaid for all transfers, and I think that the test covers all requirements but If you think that there is a problem with the scenario, I will check again with the QA team.

Reply:
I see, sure you can keep it that way. I mentioned it as the case in the test plan doesn't mention it.